tests/explicit-nulls/neg/i21629/J.java

```diff
@@ -0,0 +1,50 @@
+package javax.annotation;
+import java.util.*;
+
+public class J {
+
+    private static String getK() {
+        return "k";
+    }
+
+    @Nullable
```

Comment:
Let's also test the annotation after modifiers: `public @Nullable String s = ...`